lib/views/flight.py

```diff
@@ -147,12 +147,15 @@ class FlightSummary(RocketSummary, EnvSummary):
     x_impact: Optional[Any]
     y: Optional[Any]
     y_impact: Optional[Any]
-    # y_sol: Optional[Any]
+    y_sol: Optional[Any]
     z: Optional[Any]
     z_impact: Optional[Any]
-    # flight_phases: Optional[Any]
-    # FlightPhases: Optional[Any]
-    # TimeNodes: Optional[Any]
+    flight_phases: Optional[Any]
+    FlightPhases: Optional[Any]
+    TimeNodes: Optional[Any]
```

Comment:
these are classes, I think they should not be summarized in this model

Reply:
patched in [eb3d79a](https://github.com/RocketPy-Team/Infinity-API/pull/34/commits/eb3d79a90f706f8fff161898ea85dc41b96f431e)